app/controllers/chairs_controller.rb

```diff
@@ -0,0 +1,68 @@
+class ChairsController < ApplicationController
+  before_action :set_chair, only: %i[show edit update destroy]
+
+  # GET /chairs or /chairs.json
+  def index
+    @chairs = Chair.all
+  end
+
+  # GET /chairs/1 or /chairs/1.json
+  def show; end
+
+  # GET /chairs/new
+  def new
+    @chair = Chair.new
+  end
+
+  # GET /chairs/1/edit
+  def edit; end
+
+  # POST /chairs or /chairs.json
+  def create
+    @chair = Chair.new(chair_params)
+
+    respond_to do |format|
+      if @chair.save
+        format.html { redirect_to @chair, notice: "Chair was successfully created." }
+        format.json { render :show, status: :created, location: @chair }
```

Comment:
Was ist der Unterschied zwischen `redirect_to @chair` und `render :show`? Warum wird das eine bei der html Seite und das andere bei dem json verwendet? Bei beiden wird doch eine neue Seite gerendert, oder?

Reply:
`redirect_to` leitet die Anfrage an eine andere Website weiter, in dem Fall an` /chair/[id]`, sodass sich der dortige Controller um das Rendering kümmert. `render` führt tatsächlichen Code aus